src/input_media.rs

```diff
@@ -2,10 +2,10 @@
 
 use serde::{Deserialize, Serialize};
 
+use crate::ParseMode;
```

Comment:
Not sure whether I prefer the base level import or the specific one into the module. 🤔 

```suggestion
use crate::parse_mode::ParseMode;
```

Reply:
same with the other places :innocent: